src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { memo, useMemo } from 'react';
+import { Person } from '../../types/Person';
+import { Loader } from '../Loader/Loader';
+import { tableColumns } from './tableColumns';
+
+interface Props {
+  people: Person[];
+  selectedSlug: string | null;
+  isLoading: boolean;
+  error: string | null;
+}
+
+const TableRowComponent: React.FC<{
+  person: Person;
+  people: Person[];
+  isSelected: boolean;
+}> = ({ person, people, isSelected }) => (
+  <tr data-cy="person" className={isSelected ? 'has-background-warning' : ''}>
+    {tableColumns.map(({ key, render }) => (
+      <td key={key}>{render ? render(person, people) : person[key]}</td>
+    ))}
+  </tr>
+);
+
+TableRowComponent.displayName = 'TableRow';
+
+const TableRow = memo(TableRowComponent);
```

Comment:
move to splitted component
```suggestion
const TableRowComponent: React.FC<{
  person: Person;
  people: Person[];
  isSelected: boolean;
}> = memo(({ person, people, isSelected }) => (
  <tr data-cy="person" className={isSelected ? 'has-background-warning' : ''}>
    {tableColumns.map(({ key, render }) => (
      <td key={key}>{render ? render(person, people) : person[key]}</td>
    ))}
  </tr>
));

TableRowComponent.displayName = 'TableRow';
```